website/docs/quickstart/security.md

```diff
@@ -0,0 +1,510 @@
+'client.security.sasl.password' = 'admin-pass'
+);
+```
+
+```sql title="Flink SQL"
+CREATE CATALOG producer_catalog WITH (
+'type' = 'fluss',
+'bootstrap.servers' = 'localhost:9123',
+'client.security.protocol' = 'SASL',
+'client.security.sasl.mechanism' = 'PLAIN',
+'client.security.sasl.username' = 'producer',
+'client.security.sasl.password' = 'producer-pass'
+);
+
+```
+
+```sql title="Flink SQL"
+CREATE CATALOG consumer_catalog WITH (
+'type' = 'fluss',
+'bootstrap.servers' = 'localhost:9123',
+'client.security.protocol' = 'SASL',
+'client.security.sasl.mechanism' = 'PLAIN',
+'client.security.sasl.username' = 'consumer',
+'client.security.sasl.password' = 'consumer-pass'
+);
+```
+
+
+### Add ACLs for Producer and Consumer
+As the `admin` user, add ACLs to grant permissions:
+
+Allow `produce`r to write data:
+```sql
+-- This can used for flink 1.18 and above.
+CALL admin_catalog.sys.add_acl(
+    'cluster', 
+    'ALLOW',
+    'User:producer', 
+    'WRITE',
+    '*'
+);
+
+-- This can only used for flink 1.19 and above.
+CALL admin_catalog.sys.add_acl(
+    resource => 'cluster', 
+    permission => 'ALLOW',
+    principal => 'User:producer', 
+    operation => 'WRITE',
+    host => '*'
+);
+```
+
+Allow `consumer` to read data:
+```sql
+-- This can used for flink 1.18 and above.
+CALL admin_catalog.sys.add_acl(
+    'cluster', 
+    'ALLOW',
+    'User:consumer', 
+    'READ',
+    '*'
+);
+
+-- This can only used for flink 1.19 and above.
+CALL admin_catalog.sys.add_acl(
+    resource => 'cluster', 
+    permission => 'ALLOW',
+    principal => 'User:consumer', 
+    operation => 'READ',
+    host => '*'
+);
+```
+
+Lookup the ACLs:
+```sql
+CALL admin_catalog.sys.list_acl(
+    'cluster', 
+    'ANY',
+    'ANY', 
+    'ANY',
+    'ANY'
+);
+
+CALL admin_catalog.sys.list_acl(
+    resource => 'cluster'
+);
+```
+it will show like:
+```text title="result"
++------------------------------------------------------------------------------------------------------+
+|                                                                                               result |
++------------------------------------------------------------------------------------------------------+
+|  resourceType="fluss-cluster";permission="ALLOW";principal="User:consumer";operation="READ";host="*" |
+| resourceType="fluss-cluster";permission="ALLOW";principal="User:producer";operation="WRITE";host="*" |
++------------------------------------------------------------------------------------------------------+
+2 rows in set
+```
+
+### Create Tables Using Different Users
+Only the `admin` user can create tables:
+```sql
+CREATE TABLE `admin_catalog`.`fluss`.`fluss_order` (
+     `order_key`  INT NOT NULL,
+    `total_price` DECIMAL(15, 2),
+    PRIMARY KEY (`order_key`) NOT ENFORCED
+);
+```
+```text title="result"
+[INFO] Execute statement succeeded.
+```
+
+The `producer` user cannot create tables:
+```sql
+CREATE TABLE `producer_catalog`.`fluss`.`fluss_order1`(
```

Comment:
It maybe confusing why create table under `producer_catalog` is using `producer` user. Maybe we can seperate it into mulitple statements: 

```sql
-- switch to producer user context
USE CATALOG producer_catalog;

-- create table using producer credientials
CREATE TABLE  fluss_order1 (
...
```